backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/TagsHandler.java

```diff
@@ -75,10 +75,8 @@ public TagsHandler(Executor executor, FutureJdbi jdbi, String entityName) {
     var currentFuture =
         InternalFuture.runAsync(
             () -> {
-              for (final var tag : tags) {
-                if (tag.isEmpty()) {
-                  throw new InvalidArgumentException("Empty tag");
-                }
+              if (tags.isEmpty()) {
+                throw new InvalidArgumentException("Tags not found");
```

Comment:
if tags list are empty then it was never throw exception so fixed condition

Reply:
It was never what?

Reply:
I mean new async code was wrongly implement. it was never throw exception if tags are empty in request. so i have checked it right way here.

Reply:
But we should also check that the tags themselves aren't empty strings, right?

Reply:
Ohh yes, i will add it.

Reply:
Added validation check for each tag